sensor_msgs/PointCloud2 pcd_obj
sensor_msgs/PointCloud2 pcd_unexpobj 
string feature_type
float32 c0
float32 c1
float32 c2
---
std_msgs/Float32MultiArray stateVec
bool done
int64 steps
